src/main/java/uk/gov/ons/census/casesvc/service/RefusalService.java

```diff
@@ -36,6 +36,10 @@ public void processRefusal(
       return;
     }
 
+    if (refusalDto.getIsHouseholder() == null) {
```

Comment:
if you're going to default it to false, you can just used a primitive instead of a class